modules/configuring-quota-management-for-distributed-workloads.adoc

```diff
@@ -104,7 +104,12 @@ The cluster queue will start a distributed workload only if the total required r
 +
 [NOTE]
 ====
-In this release of {productname-short}, the only accelerators supported for distributed workloads are NVIDIA GPUs.
+. Specify a quota for each resource that the user can request (even if the requested value is 0), by updating the `spec.resourceGroups` section as follows:
+
+* Include the resource name in the `coveredResources` list.
+* Specify the resource `name` and `nominalQuota` in the `flavors.resources` section, even if the `nominalQuota` value is 0.
+
+. In this release of {productname-short}, the only accelerators supported for distributed workloads are NVIDIA GPUs.
 ====
```

Comment:
Is it correct that lines 107-111 should be within the note?  Seems to be a step

Reply:
Good point! The update started out as a simpler note, but you're right that it's now better included as part of the step. Updated accordingly, thank you!